force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -80,16 +106,33 @@ public with sharing class ClassesObjectsMethods {
      * @return A string consisting of only unique words from the input sentence.
      */
     public static String removeDuplicateWords(String sentence) {
-        // Split the sentence into words
-
-        // Create a set to store the unique words
+       // Split the sentence into words
+        String[] words = sentence.split(' ');
 
-        // Add each word to the set
+       // Create a set to store the unique words
+        Set<String> uniqueWords = new Set<String>(); 
 
-        // Join the unique words back into a sentence
+       // Add each word to the set
+        for(String word: words){
+           // Add the word to the set, if it doesn't already exist in the set.
+           // This ensures that we only add unique words to the result.
+            uniqueWords.add(word);
+        }
+       // Convert the set of unique words to a list, as required by the join method
+        List<String> uniqueWordsList = new List<String>(uniqueWords);
+
+       // Manually Join the unique words back into a sentence
+        String uniqueSentence = '';
+        for(String word : uniqueWordsList){
+            if(uniqueSentence == ''){
```

Comment:
Rather than needing this conditional block, you can add the space after each word instead of before.